src/math/function1D.cpp

```diff
@@ -337,6 +338,49 @@ const std::map<Functions1D::Form, Function1DDefinition> &functions1D()
          */
         functions[Functions1D::Form::Harmonic].setDerivativeFunction(
             [](double x, double omega, const std::vector<double> &params) { return params[0] * x; });
+
+        /*
+         * Coulombic Potential
+         *
+         * Parameters:
+         * INPUT  0 = q1
+         * INPUT  1 = q2
+         *
+         *          q1 * q2
+         * F(x)=k * -------
+         *             x
+         */
+        functions[Functions1D::Form::Coulombic] =
+            Function1DDefinition({"q1", "q2"}, [](double x, double omega, const std::vector<double> &params)
+                                 { return (COULCONVERT * params[0] * params[1]) / x; });
+        /*
+         * dYdX(x) = - k * q1 * q2 * r**-2
+         */
+        functions[Functions1D::Form::Coulombic].setDerivativeFunction(
+            [](double x, double omega, const std::vector<double> &params)
+            { return (-COULCONVERT * params[0] * params[1]) / pow(x, 2); });
```

Comment:
The `pow` function is slow for doing a simple square, so I'd suggest changing this:
```suggestion
            { return (-COULCONVERT * params[0] * params[1]) / (x * x); });
```